diagnostic.md

```diff
@@ -32,6 +32,23 @@ format described in the instructions). The page should have a title in the head
 and an unordered list (with three items) inside the body.
 
 <!-- your answer starts here -->
+```HTML
+
+<!DOCTYPE html>
+  <html>
+    <head>
+      <title></title>
```

Comment:
.5/1
Make sure you read the question carefully! It asks for you to put a title in the title, and to have text in the list items.